corehq/ex-submodules/pillowtop/management/commands/add_kafka_partition.py

```diff
@@ -29,7 +29,7 @@ def handle(self, topic, num_partitions, **options):
         )
         added_partition = raw_input("have you run {} ? [y/n]".format(kafka_command))
         if added_partition not in ['y', 'yes']:
-            print("then run it")
+            print("then run it on the kafka machine")
 
         for checkpoint in DjangoPillowCheckpoint.objects.filter(sequence_format='json'):
```

Comment:
Perhaps I'm confused about what this does but I thought the kafka pillows don't use the `DjangoPillowCheckpoint` at all anymore so the offset values in any `DjangoPillowCheckpoints` would be old. Wouldn't that result in a rewind?